js/mainScript.js

```diff
@@ -27,16 +27,12 @@ function removeBook(e, newBookElement) {
   bookArray = bookArray.filter(checkBtnclicked);
   newBookElement.remove();
 }
-let entireJSON = '';
+
+const entireJSON = localStorage.getItem('bookKey');
 function syncStorage() {
-  entireJSON = localStorage.getItem('bookKey');
   if (entireJSON != null) {
-    localStorage.setItem(
-      'bookKey',
-      entireJSON.concat(JSON.stringify(bookArray)),
-    );
+    localStorage.setItem('bookKey', entireJSON.concat(JSON.stringify(bookArray)));
```

Comment:
- [ ] Kindly make sure your local storage works properly. When I add a book and refresh the page the data isn't preserved. Also, in the local storage, I get null. Kindly fix this 🙏 .
- [ ] Kindly remove all commented code in your codebase to make your work neater 🙏 
- [ ] Kindly use ES6 format to write your `functions` this is best practice and required at Microverse 🙏

Reply:
Hi @Ugboaja-Uchechi,  ES6 format to write a function is not a project requirement.